client/src/App.js

```diff
@@ -5,18 +5,29 @@ import Login from "./pages/login";
 import Signup from "./pages/signup";
 import withAuth from "common/withAuth";
 import Campaigns from "./pages/campaigns";
+import Campaign from "./pages/campaign";
 import Prospects from "./pages/prospects";
 import Templates from "./pages/templates";
 import Reporting from "./pages/reporting";
-import { campaigns, prospects, templates, reporting, importProspects } from "constants/routes";
+import {
+  campaigns,
+  campaign,
+  prospects,
+  templates,
+  reporting,
+  importProspects,
+} from "constants/routes";
 import ImportProspects from "./pages/importProspects";
+import AuthRoute from "pages/gmailAuth/AuthRoute";
 
 function App({ isAuthenticated }) {
   return (
     <MainLayout>
       <Switch>
+        <Route path="/gmailauth" component={AuthRoute}></Route>
```

Comment:
Okay, so if we want this dialog to appear on top of content, we need to move it outside of the Switch statement, and make the path something like:
`*/gmailauth`
So that whenever the route has /gmailauth at the end of it, this dialog would show up
And if someone exits the dialog, you can just remove the /gmailauth part of hte route to return to the current page 🤗